webrtc.html

```diff
@@ -2620,8 +2620,12 @@
           </section>
         </div>
       </section>
-      <section>
+      <section class="informative">
         <h3>Legacy Interface Extensions</h3>
+          <p>This section is non-normative; it specifies methods that can be
```

Comment:
I don't think there is an appropriate existing class, but @tabatkins might know. Would you like the whole section to have a different background? You can always add your own CSS to achieve whatever effect effect is desired.